docs/develop/schema-field-descriptions-v1.0.md

```diff
@@ -7,11 +7,11 @@ title: Manifest schema v1.0
 
 # Field descriptions
 
-All fields that are not marked as **Required** are optional
+All fields that are not marked as **Required** are optional.
```

Comment:
These files are generated and update each release. Changes should not be made here to be persistent. The same for all `schema-field-descriptions-*` files